assets/pb/qm/dft/ksdft/gga/functional.nodes.yml

```diff
@@ -4,7 +4,7 @@
 pbe:
   isDefault: true
   functional:
-    name: PBE
+    title: PBE
```

Comment:
The schema has "name", doesn't it: https://github.com/Exabyte-io/esse/blob/c6fb58f3a3e17f164f2d1036cc5ba96d637cb6cf/schema/models_directory/pb/qm/dft/functional.json?